_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation-filter-popin/smart-shopping-campaign-creation-popin-dimension.vue

```diff
@@ -81,7 +80,19 @@ export default {
   computed: {
     dimensionChosen: {
       get() {
-        return this.$store.state.smartShoppingCampaigns.dimensionChosen;
+        let final = this.$store.state.smartShoppingCampaigns.dimensionChosen;
+        if (this.$store.state.smartShoppingCampaigns.dimensionChosen) {
+          final = {
+            ...this.$store.state.smartShoppingCampaigns.dimensionChosen,
+            name: this.$t(
+              `smartShoppingCampaignCreation.${this.$store.state.smartShoppingCampaigns.dimensionChosen.id}`,
+            ),
+            subtitle: this.$t(
+              `smartShoppingCampaignCreation.${this.$store.state.smartShoppingCampaigns.dimensionChosen.id}SubTitle`,
```

Comment:
As seen together, it's a problem if we don't have a computed with name and subtitle in it. Because it's a v-model it requires to be exactly the same as the data displayed so it can be already checked in case of edition